src/sentry/runner/commands/cleanup.py

```diff
@@ -177,6 +177,7 @@ def is_filtered(model):
             (models.UserReport, "date_added", None),
             (models.GroupEmailThread, "date", None),
             (models.GroupRuleStatus, "date_added", None),
+            (models.RuleFireHistory, "date_added", None),
```

Comment:
Should `RuleFireHistory` also be added to the `DIRECT_GROUP_RELATED_MODELS` list in `deletions/defaults/group.py` so that when users remove groups the rule fire history is removed too?